server/src/server.ts

```diff
@@ -230,7 +234,7 @@ function addStaticClassMembers(toReturn: CompletionItem[], item:ClassNode)
             });
 
             if (!found) {
-                toReturn.push({ label: subNode.name, kind: CompletionItemKind.Method, detail: "method (static)", insertText: subNode.name + "()" });
+                toReturn.push({ label: subNode.name, kind: CompletionItemKind.Method, detail: "method (static)", insertText: subNode.name + "(" + stringifyMethodParams(subNode) +")" });
```

Comment:
Can you remove this from this pull request please? We'll discuss this separately in #44